### PR TITLE
application attachments API

### PR DESCRIPTION
**Pull request by aurelio-aot**

### CODE REVIEW

File: applications/Unity.GrantManager/src/Unity.GrantManager.Application/Attachments/ComsS3BlobProvider.cs [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+using System.Web;
+using Unity.GrantManager.Applications;
+using Volo.Abp.BlobStoring;
+using Volo.Abp.DependencyInjection;
+
+namespace Unity.GrantManager.Attachments;
+
+public class ComsS3BlobProvider : BlobProviderBase, ITransientDependency
+{
+    private readonly IHttpContextAccessor _httpContextAccessor;
```
Comment (AndreGAot) [inline applications/Unity.GrantManager/src/Unity.GrantManager.Application/Attachments/ComsS3BlobProvider.cs:20]: Is there another ABP wrapper around this?

File: applications/Unity.GrantManager/src/Unity.GrantManager.Application/Unity.GrantManager.Application.csproj [hunk 2]
```diff
@@ -6,6 +6,7 @@
     <TargetFramework>net7.0</TargetFramework>
     <Nullable>enable</Nullable>
     <RootNamespace>Unity.GrantManager</RootNamespace>
+    <UserSecretsId>4d64e1b9-9e6e-4bb6-9dcf-b70d108adaeb</UserSecretsId>
```
Comment (AndreGAot) [inline applications/Unity.GrantManager/src/Unity.GrantManager.Application/Unity.GrantManager.Application.csproj:9]: Think can be removed

File: applications/Unity.GrantManager/src/Unity.GrantManager.HttpApi/Controllers/FileController.cs [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+                catch (Exception ex)
+                { 
+                    ErrorList.Add(source.FileName + " : " + ex.Message);
+                }
+                
+            }
+
+            if(ErrorList.Count > 0)
+            {
+                return BadRequest("ERROR:<br>" + String.Join("<br>", ErrorList.ToArray()));
```
Comment (AndreGAot) [inline applications/Unity.GrantManager/src/Unity.GrantManager.HttpApi/Controllers/FileController.cs:63]: Look at ABP exception handling

File: applications/Unity.GrantManager/src/Unity.GrantManager.Application/Unity.GrantManager.Application.csproj (hunk 2 above)
Comment (aurelio-aot) [reply applications/Unity.GrantManager/src/Unity.GrantManager.Application/Unity.GrantManager.Application.csproj]: done

File: applications/Unity.GrantManager/src/Unity.GrantManager.Application/Attachments/ComsS3BlobProvider.cs (hunk 1 above)
Comment (aurelio-aot) [reply applications/Unity.GrantManager/src/Unity.GrantManager.Application/Attachments/ComsS3BlobProvider.cs:20]: will refactor this in my next user story Ticket 8392 (Assessment Attachment)

File: applications/Unity.GrantManager/src/Unity.GrantManager.HttpApi/Controllers/FileController.cs (hunk 3 above)
Comment (aurelio-aot) [reply applications/Unity.GrantManager/src/Unity.GrantManager.HttpApi/Controllers/FileController.cs:63]: will refactor this in my next user story Ticket 8392 (Assessment Attachments)

**Review by AndreGAot — APPROVED** [review]
Approved

**Pull request merged**